jujugui/static/gui/src/app/app.js

```diff
@@ -728,6 +728,8 @@ YUI.add('juju-gui', function(Y) {
       @method _renderUserProfile
     */
     _renderUserProfile: function() {
+      var user = this.get('users')['charmstore'];
+      var username = user && user.user.name;
```

Comment:
you'll probably want to do `user && user.user && user.user.name` just in case the structure isn't properly complete